src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -54,7 +54,8 @@
             <div class="item-header d-flex">
               <!-- the filing label (left side) -->
               <div class="item-header__label">
-                <h3 class="item-header__title">{{filing.displayName}}</h3>
+                <h3 class="item-header__title"><v-icon v-if="filing.displayName==FilingNames.COURT_ORDER">mdi-gavel
+                </v-icon>{{filing.displayName}}</h3>
```

Comment:
Could you break this up into several lines, please?

```
<h3>
   <v-icon ...
   <span ...
</h3>
```

Reply:
Thanks. This looks good.

But why did you restore and space character on line 32? That should use padding instead (prob pr-1 on the icon again).

Reply:
@lambert-alex Please fix this ^^ in a future PR if needed.

Reply:
Okay I will fix this in my next pr into this repo